src/Hashvatar.elm

```diff
@@ -0,0 +1,117 @@
+module Hashvatar exposing (..)
+
+import Hash exposing (Hash)
+import Hashvatar.HexGrid as HexGrid
+import Html exposing (Html, div)
+import Html.Attributes exposing (class)
+import List.Extra as ListE
+import String.Extra exposing (break)
+import UI.Color as Color exposing (Color)
+
+
+view : Hash -> Html msg
+view hash =
```

Comment:
Its not the most efficient thing to do every time we want to render an avatar, but realistically we wont have hundreds on a page at the same time.